
orjson switched away from runtime AVX512 detection to always building for
a target that supports it. We want to continue shipping binaries that work
on older systems so we explicitly disable it again.

https://github.com/ijl/orjson/issues/603

diff -wpruN --no-dereference '--exclude=*.orig' a~/build.rs a/build.rs
--- a~/build.rs	1970-01-01 00:00:00
+++ a/build.rs	1970-01-01 00:00:00
@@ -38,7 +38,7 @@ fn main() {
     #[allow(unused_variables)]
     let is_64_bit_python = matches!(python_config.pointer_width, Some(64));
 
-    #[cfg(all(target_arch = "x86_64", not(target_os = "macos")))]
+    #[cfg(all(target_arch = "x86_64", not(target_os = "macos"), not(target_os = "illumos")))]
     if version_check::is_min_version("1.89.0").unwrap_or(false) && is_64_bit_python {
         println!("cargo:rustc-cfg=feature=\"avx512\"");
     }
